### PR TITLE
Add limits also to star solo

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
Integrates also https://github.com/galaxyproject/tools-iuc/pull/4234

File: tools/rgrnastar/rg_rnaStarSolo.xml [hunk 1]
```diff
@@ -109,6 +109,12 @@
     ## Splice junctions are always under "raw" directory
 
     --soloOutFormatFeaturesGeneField3 '${solo.soloOutFormatFeaturesGeneField3}'
+
+    ## Limits
```
Comment (bernt-matthias) [inline tools/rgrnastar/rg_rnaStarSolo.xml:113]: Can you use a token for the arguments?

Comment (bgruening) [reply tools/rgrnastar/rg_rnaStarSolo.xml:113]: No, they are both different. One is nested in a conditional and the other not.

**Review by bernt-matthias — APPROVED** [review]
OK. Do we want to rename the version token. Or is VERSION also picked up by the auto update?

File: tools/rgrnastar/rg_rnaStarSolo.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/rgrnastar/rg_rnaStarSolo.xml:113]: Might still be possible, using cheetah's getVar function https://cheetahtemplate.org/api/Cheetah.Template.html ..

File: tools/rgrnastar/macros.xml [hunk 2]
```diff
@@ -109,6 +121,7 @@
                 --genomeSAindexNbases ${refGenomeSource.genomeSAindexNbases}
             #end if
             --runThreadN \${GALAXY_SLOTS:-4}
+            --limitGenomeGenerateRAM \${GALAXY_MEMORY_BYTES}
```
Comment (wm75) [inline tools/rgrnastar/macros.xml:124]: should be possible to simplify the whole memory logic to something like:
`\$((\${GALAXY_MEMORY_MB:-31} * 1000000))`

**Comment by bgruening** [comment]
I found that old tab. Does this look better to you @wm75? The values need to be in bytes according to https://physiology.med.cornell.edu/faculty/skrabanek/lab/angsd/lecture_notes/STARmanual.pdf and the default should be 31 GB.

**Comment by bernt-matthias** [comment]
@bgruening do you want to rename the version token?

**Comment by bgruening** [comment]
> @bgruening do you want to rename the version token?

done

**Pull request closed without merging**